secant/MockedDependencies/Services.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 protocol Services {
     var networkProvider: ZcashNetworkProvider { get }
     var seedHandler: MnemonicSeedPhraseProvider { get }
-    var keyStorage: KeyStoring { get }
+    var keyStorage: WalletStorageInteractor { get }
```

Comment:
Should the name keyStorage be updated here now that we have a new type name?

Reply:
fixed